src/main/scala/circt/stage/package.scala

```diff
@@ -12,6 +12,9 @@ import java.io.File
 
 package object stage {
 
+  final val pleaseSwitchToCIRCT =
+    "this way of running Chisel is going away as part of the migration to the MLIR-based FIRRTL Compiler (MFC). Please switch to circt.stage.ChiselStage."
+
```

Comment:
I think having the more precise message is good, but can this use or share a base message with `deprecatedMFCMessage` so that users who plan to stick with Chisel 3.6 and SFC can easily suppress all of these messages with a single suppression?